doc/classes/PropertyTweener.xml

```diff
@@ -59,7 +78,25 @@
 
 				func tween_curve(v):
 				    return curve.sample_baked(v)
-				[/codeblock]
+				[/gdscript]
+				[csharp]
+				[Export]
+				public Curve Curve { get; set;}
+
+				public override void _Ready()
+				{
+				    Tween tween = CreateTween();
+				    // Interpolate the value using a custom curve.
+				    Callable tweenCurveCallable = Callable.From((float value) =&gt; TweenCurve(value));
+				    tween.TweenProperty(this, "position:x", 300.0f, 1.0f).AsRelative().SetCustomInterpolator(tweenCurveCallable);
+				}
+
+				public TweenCurve(float value)
```

Comment:
Missing return type in this method declaration, and could probably be private.

```suggestion
				private float TweenCurve(float value)
```

Reply:
For clarity, since I said I tested the code: I made a local copy of the code examples in this PR, changed things until they worked, and transcribed the changes into suggestions in this PR. I missed the return value, but it's present in the local version I tested.